### ISSUE TITLE
DASH/HLS support

### ISSUE THREAD

**Issue by chrisshiplet** [charliesbox/react-native-google-cast#47]
Are there plans to support the Google Cast library's adaptive streaming options like HLS or DASH? Looks like it's currently hardcoded to use video/mp4, and it causes a generic failure screen on the Chromecast device when I attempt to use an MPD or M3U8 playlist. If not, would you be open to a PR?

**Comment by petewilkins** [comment]
Hi @chrisshiplet, have you had any luck with implementing HLS support? I'm able to cast m3u8 files even though it's currently hardcoded for video/mp4 but I do get some strange crashes every so often which I think may be to do with the HLS stream changing quality/resolution. What was the generic failure screen you were getting?

**Comment by vishaljadav24** [comment]
Hello @chrisshiplet , I am also working on HLS streaming. Can I use this library for Cromecast HLS Streaming / m3u8 Streaming?

**Comment by chrisshiplet** [comment]
Sorry for not following up on this issue, I must've missed the notifications. The HLS problem was a serverside issue in my case, but I did end up finding the google cast sdk v2 this library uses to be extremely unreliable in general with device discovery and streaming. I ended up integrating the cast sdk v4 directly in my app. Hopefully once I understand React Native a bit more I can publish it as an npm package. I'm going to close this issue for now because HLS does technically work as well as anything else I threw at this library...
